moviepy/config.py

```diff
@@ -45,10 +45,9 @@ def try_cmd(cmd):
 else:
     success, err = try_cmd([FFMPEG_BINARY])
     if not success:
-        raise IOError(err.message +
-                 "The path specified for the ffmpeg binary might be wrong")
-
-
+        raise IOError(
+            str(err) +
+            " - The path specified for the ffmpeg binary might be wrong")
```

Comment:
According to [this](https://stackoverflow.com/questions/33239308/how-to-get-exception-message-in-python-properly), some built-in exceptions have a msg attribute, some don't.

IOError's definition is platform dependent. When I look at builtins.py on Windows, Python 3.6, it isn't present. Maybe it is on Linux?